test/e2e/fixtures/threebox-enabled/state.json

```diff
@@ -97,7 +97,7 @@
       "tokens": [],
       "hiddenTokens": [],
       "suggestedTokens": {},
-      "useBlockie": true,
+      "useBlockie": false,
```

Comment:
I don't believe we need this change to the threebox fixture

Reply:
Yes, you're correct. I fixed it.